app/src/Containers/SearchContainer.tsx

```diff
@@ -241,31 +240,35 @@ const SearchContainer = () => {
     return undefined
   }
 
+  
   const handleQuickSearchSubmit = (
     searchField: 'client' | 'description' | 'name' | 'role' | 'skills' | 'sector',
     searchQueryChoice: string,
   ) => {
     let searchQueries = [] as string[]
+    let results = [] as Projects
+
+    searchQueries.push(searchQueryChoice)
 
     if (searchField === 'role') {
       const relatedRoles = getRelatedRoles(searchQueryChoice)
 
       if (relatedRoles?.length) {
         searchQueries = relatedRoles
       }
+      results = fuzzySearchByArray(searchQueries, [searchField]) // we need to use fuzzy search as the roles should include various options of a role (i.e. without it developer does not include web developer results) 
```

Comment:
It would be great if we can make this a little clearer - I think the core of the reason for using fuzzy search here is that role names are not exact (different charities use plenty of different ways of naming them), unlike tech stacks or sectors where it's reasonable to assume we can look for an exact match

Reply:
Thanks Joe, hope it's more clear now.